_format/1.1/index.md

```diff
@@ -651,25 +817,26 @@ values.
 not be represented as a single concatenated string with its values separated by
 whitespace as might be the case in a [`Link` header serialization](https://tools.ietf.org/html/rfc8288#section-3.5).
 
-#### <a href="#profile-links" id="profile-links" class="headerlink"></a> Profile Links
+#### <a href="#extension-and-profile-links" id="extension-and-profile-links" class="headerlink"></a> Extension and Profile Links
 
-Like all [links][link], a link in an array of `profile` links can be represented
-with a [link object].
+Top-level links **MAY** contain the `ext` and `profile` members. The values of
+these members **MUST** be an array of [links][link]. Like all links, each can be
+represented with a string or a [link object].
 
-Here, the `profile` key specifies an array of `profile` links:
+Here is an example that includes links to both an extension and profiles:
 
 ```json
 "links": {
+  "ext": [
+    "https://jsonapi.org/ext/atomic"
+  ],
   "profile": [
     "http://example.com/profiles/flexible-pagination",
     "http://example.com/profiles/resource-versioning"
   ]
 }
 ```
 
-> Note: Additional link types, similar to `profile` links, may be specified in
-the future.
-
```

Comment:
I think this entire section needs to be removed since we no longer have `ext` or `profile` links defined above, especially since the example shows and array of links.

Reply:
Before removing, let's confirm that we really don't want to allow an in-document representation of which profiles / extensions have been applied.

Reply:
Discussed with @gabesullice and decided to move these URI arrays (`ext` and `profile`) as optional members of the `jsonapi` object.